managers/oclcClassify.py

```diff
@@ -18,7 +18,7 @@ class ClassifyManager:
     Returns:
         [str] -- A string of XML data comprising of the Classify response body.
     """
-    CLASSIFY_ROOT = 'https://metadata.api.oclc.org/classify'
+    CLASSIFY_ROOT = 'https://metadata.api.oclc.org/classify/'
```

Comment:
It might be worth s code comment explaining that this slash is necessary for the endpoint to work!

Reply:
Agreed